src/Routing/TaxonomyChecksRouteSubscriber.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\workbench_access\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Routing\Route;
+
+/**
+ * Class TaxonomyChecksRouteSubscriber.
+ *
+ * @package Drupal\anl_taxonomy_checks\Routing
```

Comment:
Remove

Reply:
@agentrickard Are you saying I should remove the @package tag or correct the value of the @package tag?

Reply:
Remove the @package tag. But see my longer notes first.

Reply:
I've removed this class, it's not needed.